run.sh

```diff
@@ -67,6 +67,8 @@ else
         NGINX_REWRITE='rewrite ^/liquidregtest(/.*)$ $1 break;'
         NGINX_REWRITE_NOJS='rewrite ^/liquidregtest(/.*)$ " /liquidregtest/nojs$1?" permanent'
         NGINX_NOSLASH_PATH="liquidregtest"
+
+        ELECTRS_ARGS="$ELECTRS_ARGS --jsonrpc-import"
```

Comment:
It would be nicer if the magic bytes were configurable so the `--jsonrpc-import` workaround wasn't necessary, but I think its fine for now (and it also has the side benefit of being faster for small regtest chains, compared to reading from the blk files).

I would only suggest to add a comment here explaining why `--jsonrpc-import` is used.

Reply:
comment added in amendment.